src/main/java/kubatech/mixin/mixins/InfernalMobs/InfernalMobsCoreAccessor.java

```diff
@@ -0,0 +1,57 @@
+package kubatech.mixin.mixins.InfernalMobs;
+
+import java.util.ArrayList;
+
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.item.ItemStack;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+import atomicstryker.infernalmobs.common.InfernalMobsCore;
+import atomicstryker.infernalmobs.common.mods.api.ModifierLoader;
+
+@Mixin(value = InfernalMobsCore.class, remap = false)
+public interface InfernalMobsCoreAccessor {
```

Comment:
Since we have source of Infernal Mobs, making these classes public on source code makes more sense.

Reply:
https://github.com/GTNewHorizons/Infernal-Mobs/pull/13

Reply:
Done